pytest_fixtures/component/katello_certs_check.py

```diff
@@ -0,0 +1,79 @@
+# katello_certs_check Fixtures
+from pathlib import Path
+
+import pytest
+from box import Box
+from broker import Broker
+from fauxfactory import gen_string
+
+from robottelo.hosts import ContentHost
+
+
+@pytest.fixture(scope='module')
+def cert_data():
```

Comment:
No setup happening here, its just a dictionary. It could be a module variable - doesn't even need to be a function.